translate/cloud-client/translate_v3_translate_text.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START translate_v3_translate_text]
+from google.cloud import translate
+
+
+def sample_translate_text(
+    text="Hello, world!", target_language="fr", project_id="[Google Cloud Project ID]"
```

Comment:
Thoughts about inlining `text` and `target_language`

below would be:
```
response = client.translate_text(
        parent=parent,
        contents=["Hello, world!"],
        mime_type="text/plain",  # mime types: text/plain, text/html
        source_language_code="en-US",
        target_language_code="fr",
    )
```

Reply:
Yes, that's simpler.